<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BPMNormalization</class>
 <widget class="QWidget" name="BPMNormalization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BPMNormalization</string>
  </property>
  <widget class="QDoubleSpinBox" name="baseBPM">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>2147483647.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Base BPM</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="origTiming">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Paste timing points here:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="normSV">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>270</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Results:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="normalize">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Normalize!</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
